backend/src/test/java/capstone/myfinancemanager/manager/service/UserServiceImpTest.java

```diff
@@ -0,0 +1,68 @@
+package capstone.myfinancemanager.manager.service;
+
+import capstone.myfinancemanager.manager.exceptions.PasswordNotMatchException;
+import capstone.myfinancemanager.manager.exceptions.UserExistsException;
+import capstone.myfinancemanager.manager.model.User;
+import capstone.myfinancemanager.manager.respository.UserRepo;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+import java.util.Optional;
+
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.*;
+
+class UserServiceImpTest {
+    private final UserRepo userRepo = mock(UserRepo.class);
+    private final UserServiceImp userServiceImp = new UserServiceImp(userRepo);
+    private final User expectedTestUser = userServiceImp.addNewUser("testuser@test.com", "testuser", "123456789");
+    private final User newUser = new User("testuser@test.com", "testuser", "98765431", "98765431", null);
+    private final User userWithNotMatchPassword = new User("testuser@test.com", "testuser", "98765431", "987654391", null);
+
+    @Test
+    void shouldCreateUser() {
+        //When
+        when(userRepo.save(any(User.class))).thenReturn(expectedTestUser);
```

Comment:
Ich würde anstatt any lieber ein konkretes objekt benutzten, dafür musst nur mit dem timestamp gucken. Ich würde dafür das generieren von dem timestampt (UserServiceImp.class Zeile 38) auslagern in eine eigene Klasse und dann kannst du das in den Tests mocken.

Reply:
ich habe das nicht richtig verstanden. Also quasi ein dummy Date festsetzen und dann testen?? 
wenn das nicht so wichtig ist würde ich das lieber später machen? sonst bitte kurz treffen ?